src/client/rooms/PrivateRoomContainer.tsx

```diff
@@ -23,6 +25,17 @@ export const PrivateRoomContainer: React.FunctionComponent<Props> = (
         }
     );
 
+    const componentDidMount = React.useCallback(
+        (socket: SocketIOClient.Socket) => {
+            return socket.emit(RoomEvent.PRIVATE_ROOM_JOIN, {
+                sessionId: props.match.params.roomId,
+            });
+        },
+        [props.match.params.roomId]
+    );
+
+    useSocket("", undefined, componentDidMount);
```

Comment:
should this be ""?

Reply:
Yeah, it was a hack to not listen on anything. Ill change this hack tho